### PR TITLE
CVE-2020-10689 init.

### PR DESCRIPTION
**Pull request by guitos**

### CODE REVIEW

File: 2020/10xxx/CVE-2020-10689.json [hunk 1]
```diff
@@ -4,15 +4,73 @@
...
+                "refsource": "MISC"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A flaw was found in the Eclipse Che up to version 7.8.x, where it did not properly restrict access to workspace pods. An authenticated user can exploit this flaw to bypass JWT proxy and gain access to the workspace pods of another user. Successful exploitation requires knowledge of the service name and namespace of the target pod."
```
Comment (cristina479) [inline 2020/10xxx/CVE-2020-10689.json:62]: The version in the description and in the version_value field do not match.

Comment (guitos) [reply 2020/10xxx/CVE-2020-10689.json:62]: The flaw was fixed in version 7.9.0 and the affected versions are 7.8.x, why they didn't match?

Comment (cristina479) [reply 2020/10xxx/CVE-2020-10689.json:62]: The 'version_value' field is inside the 'affects' object. This means that 7.9.0 is the affected version unless the 'version_value' field explicitly states that 7.9.0 is the fixed version. Currently it reads as if 7.9.0 is the affected version.

Comment (guitos) [reply 2020/10xxx/CVE-2020-10689.json:62]: The version_value inside the affects object was updated to reflect the version in the description.
Thank you for the clarification!

**Pull request merged**